static/js/domain-registration.js

```diff
@@ -189,7 +189,7 @@
                 modalRegistrationSuccessState.classList.remove("is-hidden");
 
                 const domainPreview = document.querySelector(".js-premium-onboarding-domain-registration-preview");
-                domainPreview.textContent = domain + ".mozmail.com";
+                domainPreview.textContent = domain;
```

Comment:
Removed ".mozmail.com" here since I'm adding that string directly into the HTML.